### ISSUE TITLE
Some issues with html2canvas

### ISSUE THREAD

**Issue by JamesJGoodwin** [niklasvh/html2canvas#1256]
Hello. It's really a pleasure for me to have a plugin like this. But there's some issues I'd like to report

1) html2canvas not working in latest Firefox
2) Generated images are blurry and sometimes cropped

I'm using 0.5.0-beta-4, btw. Can't wait till version 1.0 released.

Regards,
Andrew.

**Comment by niklasvh** [comment]
Is this still an issue with [v1.0.0](https://github.com/niklasvh/html2canvas/releases)? If so, could you please share an example on [jsfiddle](http://jsfiddle.net/az5j0766/).

**Comment by JamesJGoodwin** [comment]
@niklasvh oh, now screenshots are not blurry anymore. But `<svg>` tags are still processing incorreclty. Here, take a look:

Screenshot made with Gyazo. That's how block should look like:

![image-default](https://user-images.githubusercontent.com/17240507/33890710-afd5dd0e-df5c-11e7-85ca-1f16b72323e0.png)

But for the first time I'm keep getting this:

![91a57ae85afe89e84c1bfa4dad7142e2](https://user-images.githubusercontent.com/17240507/33892181-d434de8a-df60-11e7-827c-5f8be13e04cc.png)

That's because it have a transparency(idk why, default option for `background` is `#fff`. Even if I specify `background: #fff` it'll still have half-transparent). But, if I try to re-render it a few times it becomes OK except `<svg>` in the top right corner:

![image-html2canvas](https://user-images.githubusercontent.com/17240507/33892293-1d99c5ea-df61-11e7-9418-0df183a6e007.png)

Btw, the card I'm trying to make a screenshot of was dynamically added on the page if it helps you to solve the issue.

**Comment by niklasvh** [comment]
Does the background issue fix itself if you set `backgroundColor: #ffffff` in the options with 1.0.0-alpha.4?

**Comment by JamesJGoodwin** [comment]
@niklasvh yes, updating to 1.0.0-alpha.4 has fixed the issue with background color. `<svg>` is still rendering incorrectly. 

![default](https://user-images.githubusercontent.com/17240507/33892981-e8249be0-df62-11e7-9dcb-7916a55a34e6.png)

**Comment by niklasvh** [comment]
Would you be able to replicate the SVG issue on jsfiddle so I could take a look?

**Comment by JamesJGoodwin** [comment]
@niklasvh sure, take a look: https://jsfiddle.net/khqgasLz/
